submissions/hornwitser_dissector/wireshark-dissector.md

```diff
@@ -0,0 +1,49 @@
+## Wireshark Dissector for Factorio <author>Hornwitser</author>
+
+This is a project that started around the Gridlock Cluster.
+I was investigating some issues with clients randomly dropping from the game and that brought me into examining packet traces using Wireshark.
```

Comment:
Maybe explain shortly what `packet traces` are.

Reply:
I expanded a bit on this section.